.cirrus.yml

```
@@ -94,7 +95,8 @@ task:
       - IMAGE: python:3.7-slim
       - IMAGE: python:3.8-slim
       - IMAGE: python:3.9-slim
-      - IMAGE: python:3.10-rc-slim
+      - IMAGE: python:3.10-slim
```

Since we are using `rc-slim` below to always get the latest dev. version, why not use `3-slim` here and only use major version numbers for the older entries?

I think this would be more coherent, and has the advantage of always tracking the latest stable Python version even if we forgot to update the CI config.

```suggestion
      - IMAGE: python:3-slim
```

---

I like specifically listing the major versions. We should also list the development head, though.

---

I guess the Linux and Windows CI are fast enough the duplicated work shouldn't be an issue.

---

well, over the course of the year, the dev version will diverge from the last release. Testing against it gives us a heads up if a future change is going to break something.

Extremely unlikely for `ppb-vector`, but we might as well.

---

Oh, I think we misunderstood one-another. I wasn't suggesting not to test against `rc-slim` or such, I was suggesting that we should refer to the latest release as just `3-silm` (which always points to the latest stable release, AFAIK) so if we forget to update the version number we are still testing against the latest stable anyhow.